website/data/commands-nav-data.json

```
@@ -412,7 +412,15 @@
   },
   {
     "title": "monitor",
-    "path": "monitor"
+    "routes": [
+      {
+        "title": "monitor",
```

```suggestion
        "title": "Overview",
```

change to match how we label section index pages. And thanks for merging the docs directory changes!